test/common/testlib.py

```diff
@@ -2494,16 +2494,19 @@ def setup_ssh_auth(self) -> None:
         for m in self.machines:
             self.authorize_pubkey(self.machines[m], "admin", pubkey)
 
-    def add_ram_disk(self, size: int = 50) -> str:
+    def add_ram_disk(self, size: int = 50, delay: Optional[int] = 0) -> str:
         """Add per-test RAM disk
 
         The disk gets removed automatically when the test ends. This is safe for @nondestructive tests.
 
+        Optionally takes a delay in nanoseconds which delays IO respones.
```

Comment:
Please also add some recommended values here, like you did in the commit message. I suppose in most cases you want a reasonably "slow" disk, so the "100000000 is ~ 40 kB/s" is useful info.